### ISSUE TITLE
Linebreaking in links

### ISSUE THREAD

**Issue by MiroslavMikus** [jesseduffield/lazygit#1270]
Not sure if this is a known issue...

**Describe the bug**
When clicked on some link with linebreak, the destination is just the first part of the link.

**Expected behavior**
The target destination should be the full link.

**Screenshots**
![image](https://user-images.githubusercontent.com/29312172/114581273-db5cbe00-9c7f-11eb-9f0a-0a75d56fd870.png)

**Desktop:**
OS: Windows 10.0.19042
Terminal: WindowsTerminal 1.6.10571.0
Powershell: 7.1.3
Lazygit Version 0.27.4

**Additional context**

**Comment by jesseduffield** [comment]
yeah it would be good to fix wrapping in general, though I also think it would be cool to just use a hyperlink (of course this too could be broken by bad wrapping).
